Sources/Testing/Events/Recorder/Event.HumanReadableOutputRecorder.swift

```diff
@@ -366,18 +404,21 @@ extension Event.HumanReadableOutputRecorder {
       let testData = testDataGraph?.value ?? .init(startInstant: instant)
       let issues = _issueCounts(in: testDataGraph)
       let duration = testData.startInstant.descriptionOfDuration(to: instant)
+      let testCasesCountMessage = _includeNumberOfTestCasesIfNeeded(test.isParameterized,
```

Comment:
Here, you can probably just write:

```swift
let testCasesCount = if verbosity >= 2 && test.isParameterized {
  " with \(testData.testCasesCount.counting("test case"))"
} else {
  ""
}
```

("Message" has a domain-specific meaning in this file, which is why I renamed this variable in my example above.)